README.md

```diff
@@ -908,6 +908,26 @@ word-break:break-word
      </td>
      <td>Read</td>
    </tr>
+  <tr>
+     <td><code>bigNumericFieldsPrecision</code>
+     </td>
+     <td>A config to specify the precision to use while reading all the BigNumeric fields of a table.
+          Please note that there might be data loss if the actual data's precision is more than what is specified.
+          By default this is not set and precision from the field is used.
+         <br/> (Optional)
+     </td>
+     <td>Read</td>
+   </tr>
+  <tr>
+     <td><code>bigNumericFieldsScale</code>
+     </td>
+     <td>A config to specify the scale to use while reading all the BigNumeric fields of a table.
```

Comment:
Should be `An alternative default scale for BigNumeric fields. Values can be between 0 and 38, and less then bigNumericFieldsPrecision.  This default is used only when the field has an unparameterized BigNumeric type.`

Reply:
Fixed